src/ShareToken.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { db } from './lib/firebase';
+import { useHistory } from 'react-router-dom';
+import { withSnackbar } from 'notistack';
+
+const ShareToken = (props) => {
+  const [shareToken, setShareToken] = useState(null);
+
+  // 1. Make a on submit function where we pull list of collections
+  let history = useHistory();
+  const onSubmitHandler = (event) => {
+    event.preventDefault();
+    db.collection(shareToken)
+      .get()
+      .then((querySnapshot) => {
+        if (querySnapshot.empty) {
+          props.enqueueSnackbar('Incorrect token, please try again', {
+            variant: 'error',
+          });
+        } else {
+          props.enqueueSnackbar('Success', {
+            variant: 'success',
+          });
+          localStorage.setItem('userToken', shareToken);
```

Comment:
In the future, we should add a function here that passes the token value to the `App` component:
```props.setToken(token);```
I had to do it in our feature to make sure `App` starts fetching data from firebase right away if that makes sense.

Reply:
Yea that totally makes sense! Maybe we could do it during the merge conflict or would it be better to create a new issue? Is there such thing as a change being too small for it to be associated with an issue?